modules/api/src/main/java/io/fluo/api/config/FluoConfiguration.java

```diff
@@ -667,4 +672,66 @@ public static void setDefaultConfiguration(Configuration config) {
     config.setProperty(MINI_START_ACCUMULO_PROP, MINI_START_ACCUMULO_DEFAULT);
     config.setProperty(MINI_DATA_DIR_PROP, MINI_DATA_DIR_DEFAULT);
   }
+  
+  public FluoConfiguration setPositiveInt(String property, int value) {
```

Comment:
`FluoConfiguration` already has a lot of implementation methods that are included by extending `CompositeConfiguration` so I thought it was OK to include as part of the API.  However, they don't need to be used by users so I agree that it's probably better to make them private.  The class already has a ton of methods.